package.json

```diff
@@ -17,18 +17,24 @@
     "hana",
     "hdb"
   ],
-  "scripts": {},
+  "scripts": {
+    "lint": "lab test -dL"
+  },
   "dependencies": {
-    "seneca": "0.5.11",
-    "hdb": "0.2.0",
-    "underscore": "1.4.x",
+    "seneca": "plugin",
+    "hdb": "0.5.3",
+    "lodash": "~4.x.x",
```

Comment:
so, you replaced underscore with lodash? you need to replace it in every file where it is used then